### PR TITLE
fix/ui bug fixes

### PR DESCRIPTION
**Pull request by alexworker23**
In this PR: 
- Added extended error messages for all api handlers by providing error.message received from backend
- Fixed Direct Buy form validation
- Fixed not displaying PartiallyFilled asks and bids in My Orders tables
- Added proper pending status tracking for blockchain transaction in Blockchain Inbox
- Fixed displaying Success Icon in some error notifications

### CODE REVIEW

File: packages/ui/libs/organization/data/src/members.ts [hunk 1]
```diff
@@ -96,18 +97,12 @@ export const useOrganizationMemberRoleUpdate = () => {
...
-            showNotification(
-              t('organization.members.notifications.roleChangeError'),
-              NotificationTypeEnum.Success
-            );
-          }
+          showNotification(
+            `${t('organization.members.notifications.roleChangeError')}:
+            ${error?.response?.data?.message || ''}
+            `,
+            NotificationTypeEnum.Success
```
Comment (josipbagaric) [inline packages/ui/libs/organization/data/src/members.ts]: This looks like an error but it returns success?

File: packages/ui/libs/user/data/src/handlers/adminUpdateUser.ts [hunk 2]
```diff
@@ -32,9 +32,11 @@ export const useAdminUpdateUser = (id: string) => {
...
           navigate('/admin/users');
         },
-        onError: () => {
+        onError: (error: any) => {
           showNotification(
-            t('admin.updateUser.notifications.userUpdateError'),
+            `${t('admin.updateUser.notifications.userUpdateError')}:
+            ${error?.response?.data?.message || ''}
+            `,
             NotificationTypeEnum.Success
```
Comment (josipbagaric) [inline packages/ui/libs/user/data/src/handlers/adminUpdateUser.ts]: Same here.

File: packages/ui/libs/certificate/data/src/handlers/blockchainTransferCertificate.ts [hunk 3]
```diff
@@ -37,16 +39,19 @@ export const useBlockchainTransferCertificateHandler = (
...
       if (receipt.status === 0) {
         throw new Error('Transaction failed');
+      } else {
+        setTxPending(false);
+        showNotification(
+          t('certificate.blockchainInbox.notifications.transferSuccess'),
+          NotificationTypeEnum.Success
+        );
+        queryClient.resetQueries(blockchainCertificatesQueryKey);
+        resetList();
```
Comment (josipbagaric) [inline packages/ui/libs/certificate/data/src/handlers/blockchainTransferCertificate.ts:53]: Should we also show Error notification if `receipt.status == 0`?

Comment (alexworker23) [reply packages/ui/libs/certificate/data/src/handlers/blockchainTransferCertificate.ts:53]: it is shown below in `catch`

**Review by josipbagaric — APPROVED** [review]

**Pull request merged**